aten/src/ATen/native/cudnn/Conv.cpp

```diff
@@ -323,7 +323,7 @@ struct ConvolutionArgs {
 
 // Hashing machinery for ConvolutionParams
 struct ParamsHash {
-  size_t operator()(const ConvolutionParams& params) const {
+  std::size_t operator()(const ConvolutionParams& params) const {
```

Comment:
@goldsborough got rid of the `std::` qualifiers. Does the HIP compiler need the qualifier here? Does including `<cstddef>` help here?